### PR TITLE
[Security][Validator] Add translations for Tagalog

### PR DESCRIPTION
**Pull request by ergiegonzaga**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes 
| BC breaks?    | no
| Deprecations? |no 
| Tests pass?   |no
| Fixed tickets | #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

**Comment by ergiegonzaga** [comment]
Why failed?

**Comment by javiereguiluz** [comment]
@ergiegonzaga there is a minor problem in one of the XML files. This is how you can solve it:

1) Open the `src/Symfony/Component/Validator/Resources/translations/validators.tl.xlf` file.
2) Remove the `+` contained at the beginning of each line.
3) Save the changes, commit the changes (`git commit src/Symfony/Component/Validator/Resources/translations/validators.tl.xlf`) and push the changes.

**Comment by ergiegonzaga** [comment]
Ahh okay sir , thanks

**Comment by ergiegonzaga** [comment]
Sir? why does it failed again this time? what's the problem?

**Comment by javiereguiluz** [comment]
@ergiegonzaga there are some problems which are unrelated to your changes ... but there's still a super minor problem with your pull request. The last line of the `validators.tl.xlf` file contains an extra `+` at the beginning of the line.

**Comment by ergiegonzaga** [comment]
okay sir i'll look at it now.

**Comment by ergiegonzaga** [comment]
Sir, why does it failed again? I already removed the last (+) sign.

**Comment by ergiegonzaga** [comment]
Hi! can someone help me on why does my pull request fails ? 
 Thank you.

**Comment by ergiegonzaga** [comment]
Hi! can someone help me on why does my pull request fails ?
Thank you.

File: src/Symfony/Component/Validator/Resources/translations/validators.tl.xlf [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+                <source>A PHP extension caused the upload to fail.</source>
+                <target>Ang dahilan ng pagkabigo ng pagupload ng files ay isang extension ng PHP.</target>
+            </trans-unit>
+            <trans-unit id="54">
+                <source>This collection should contain {{ limit }} element or more.|This collection should contain {{ limit }} elements or more.</source>
+                <target>Ang koleksyon na ito ay dapat magkaroon ng {{ limit }} elemento o marami pa.|Ang koleksyon na ito ay dapat magkaroon ng {{ limit }} mga elemento o marami pa.</target>
+            </trans-unit>
+            <trans-unit id="55">
+                <source>This collection should contain {{ limit }} element or less.|This collection should contain {{ limit }} elements or less.</source>
+                <target>Ang koleksyon na ito ay dapat magkaroon ng {{ limit }} elemento o maliit pa.|Ang koleksyon na ito ay dapat magkaroon ng {{ limit }} mga elemento o maliit pa.<</target>
```
Comment (eXtreme) [inline src/Symfony/Component/Validator/Resources/translations/validators.tl.xlf]: @ergiegonzaga remove extra `<` from `<</target>`

Comment (ergiegonzaga) [reply src/Symfony/Component/Validator/Resources/translations/validators.tl.xlf]: ahh, thanks sir, I did't really see that one.

**Comment by ergiegonzaga** [comment]
Someone help me please with the error on my pull request..

**Comment by nicolas-grekas** [comment]
@ergiegonzaga good news, I rebased and squashes your PR on latest master (using the `git rebase -i` command) and this is now green.

**Comment by fabpot** [comment]
@nicolas-grekas Translations should always be merged in the lowest maintained branch (2.7 right now) even if some translations don't exist in 2.7.

**Review by nicolas-grekas — APPROVED** [review]
for 2.7

File: src/Symfony/Component/Security/Core/Resources/translations/security.tl.xlf [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                <target>Ang Cookie ay ginamit na ng ibang tao.</target>
+            </trans-unit>
+            <trans-unit id="6">
+                <source>Not privileged to request the resource.</source>
+                <target>Walang pribilehiyo upang humingi ng mga bagong mapagkukunan.</target>
+            </trans-unit>
+            <trans-unit id="7">
+                <source>Invalid CSRF token.</source>
+                <target>Hindi balidong mga token ng CSRF.</target>
+            </trans-unit>
```
Comment (chalasr) [inline src/Symfony/Component/Security/Core/Resources/translations/security.tl.xlf]: number 8 is missing https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Security/Core/Resources/translations/security.en.xlf#L33 , required for 2.7

**Review by chalasr — CHANGES_REQUESTED** [review]

**Comment by ergiegonzaga** [comment]
Hi sir,I already updated the security.tl.xlf and added trans #8 .

File: src/Symfony/Component/Security/Core/Resources/translations/security.tl.xlf (hunk 2 above)
Comment (ergiegonzaga) [reply src/Symfony/Component/Security/Core/Resources/translations/security.tl.xlf]: Sir, I already added the missing trans-unit #8 ...

**Comment by ergiegonzaga** [comment]
sir? why does the (continuous-integration/appveyor/pr ) failed ?

**Comment by ergiegonzaga** [comment]
Please someone, help me with the error..

**Comment by eXtreme** [comment]
@ergiegonzaga looking at other pending PRs, this failure seems not to be related to your changes so you shouldn't be worried.

**Comment by ergiegonzaga** [comment]
aww okay sir, ill just wait for the project owner to review my pull request..

**Comment by fabpot** [comment]
You did great! Thank you @ergiegonzaga.

**Comment by ergiegonzaga** [comment]
Thank you sir, what other files I could translate into tagalog sir?

**Comment by fabpot** [comment]
There are some more translations under `src/Symfony/Component/Form/Resources/translations/`.

**Pull request closed without merging**